pageserver/src/http/routes.rs

```diff
@@ -1029,6 +1029,16 @@ async fn active_timeline_of_active_tenant(
         .map_err(ApiError::NotFound)
 }
 
+async fn always_panic_handler(_: Request<Body>) -> Result<Response<Body>, ApiError> {
+    // catch_unwind ensures that tokio nor hyper are distracted by our panic, but panic hooks
+    // (at least sentry) will report the issue. if tested across fleet, this should be a good test
+    // case for panic deduplication as well per version.
+    let _ = std::panic::catch_unwind(|| {
+        panic!("unconditional panic for testing panic hook integration")
```

Comment:
How about repeating a caller-provided string in the panic message?
Caller can still use a constant, but that way, they can pass in a nonce and check the logs / sentry for that exact nonce.

Reply:
I'll add an optional message query parameter.

Reply:
I'll just dump the whole url decoded query actually.

Reply:
[9d8e796](https://github.com/neondatabase/neon/pull/3475/commits/9d8e796e866ed4e6f5b9f2f4ab98f00b6ec0c47d)